server/container_create.go

```diff
@@ -519,6 +521,8 @@ func (s *Server) CreateContainer(ctx context.Context, req *types.CreateContainer
 			log.Errorf(ctx, "CreateCtr: failed to save progress of container %s: %v", newContainer.ID(), err)
 		}
 		log.Infof(ctx, "CreateCtr: context was either canceled or the deadline was exceeded: %v", ctx.Err())
+		// no need to cleanup
```

Comment:
sorry this is not `no need to cleanup` but rather `should not cleanup` as we've explicitly stored it to return to later

Reply:
processed